network-api/networkapi/templates/pages/buyersguide/base.html

```diff
@@ -78,22 +78,25 @@
   {% include "fragments/buyersguide/primary_nav.html" %}
   </div>
 
+  {% get_bg_home_page as home_page %}
+
   {% block hero %}{% endblock %}
 
+  {% block category_header %}{% endblock %}
+
   {% block category_nav %}
-  {% get_bg_home_page as home_page %}
   {% include "fragments/buyersguide/pni_mobile_nav.html" with pagetype=pagetype categories=categories current_category=current_category %}
   <nav id="multipage-nav" class="pni-category-nav text-center d-none d-md-block tw-no-scrollbar tw-mt-2" title="{% trans "site navigation" context "Tooltip on menu items" %}">
-    <div class="container" id="product-review">
+    <div class="container tw-py-2" id="product-review">
       <div class="row">
         <div class="col">
           {% if pagetype == "product" or pagetype == "about" %}
-            <a class="multipage-link active" data-name="None" data-mobile="{% trans "All Categories" %}" href="{% relocalized_url home_page.localized.url %}">{% trans "All" %}</a>
+            <a class="multipage-link active" data-name="None" data-mobile="{% trans "All Categories" %}" href="{% relocalized_url home_page.localized.url %}">{% trans "All Reviews" %}</a>
           {% else %}
-            <a class="multipage-link {% if not category %} active{% endif %}" data-name="None" data-mobile="{% trans "All Categories" %}" href="{% relocalized_url home_page.localized.url %}">{% trans "All" %}</a>
+            <a class="multipage-link {% if not category %} active{% endif %}" data-name="None" data-mobile="{% trans "All Categories" %}" href="{% relocalized_url home_page.localized.url %}">{% trans "All Reviews" %}</a>
           {% endif %}
 
-          {% for cat in categories %}
+          {% for cat in categories|slice:":6" %}
```

Comment:
No idea needs to be tested more in staging

Reply:
Is there maybe a more robust way to detect if this fits? CSS or JS based maybe? Since we are not in charge of the content and with the variation between all the languages I think this might be hard to test.

Reply:
Honestly to address the feedback for the tablet solution we need to do a solution like you mentioned. I think the acceptance criteria of how many categories should probably be updated to reflect this as well but I will try to think of something!